.chloggen/purefa-receiver-missing-scrapers.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receiver/purefareceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Apparently, the non-array metrics are not currently working as expected. 
```

Comment:
The note should reflect the fix that was made.  Something like "Fixed issue where non-array metrics were not working as expected" but try to specify how they were not working.